qiskit/providers/ibmq/api_v2/clients/websocket.py

```diff
@@ -130,7 +136,11 @@ def _connect(self, url):
         return websocket
 
     @asyncio.coroutine
-    def get_job_status(self, job_id, timeout=None):
+    def get_job_status(
+            self,
+            job_id: str,
+            timeout: Optional[int] = None
```

Comment:
The timeout value is actually `float`, not `int`. Can you also correct the docstring?